src/app/productConfig/productConfig.component.js

```diff
@@ -46,7 +46,6 @@ export default angular
       campaignCode: '@',
       campaignPage: '@',
       productCode: '@',
-      buttonLabel: '@',
```

Comment:
This will break the AEM configuration.

Reply:
Thanks for noticing that. I removed that commit.